packages/module/src/module.ts

```diff
@@ -1,4 +1,5 @@
 import { readdirSync } from 'node:fs'
+import { join } from 'node:path'
```

Comment:
Thanks for PR, 

Can you change it from `node:path` to [`pathe`](https://github.com/unjs/pathe/blob/main/src/path.ts#L62) 
Since it's a dependency in `nuxt` and `nuxt/kit`, we can use it instead to prevent path problems

Reply:
@sadeghbarati done! 

And thank you for the amazing project.

Reply:
It's Community / Team Work Project 🙏 🤩